sdk/core/Azure.Core.Experimental/CHANGELOG.md

```diff
@@ -1,16 +1,14 @@
 # Release History
 
-## 0.1.0-preview.26 (Unreleased)
+## 0.1.0-preview.26 (2023-04-10)
 
 ### Features Added
 
-### Breaking Changes
+- Added basic debugger support for DynamicData
 
 ### Bugs Fixed
 
-- Use specified `DynamicJsonNameMapping` in array and object enumerators.
-
-### Other Changes
+- Use `DynamicJsonOptions` in child elements, arrays, and object enumerators.
```

Comment:
```suggestion
- Use root `DynamicJsonOptions` in child elements, arrays, and object enumerators.
```

Reply:
I think you added this in your other comment :)